[2022-08-02 16:20:14,099] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-02T16:10:00+00:00 [queued]>
[2022-08-02 16:20:14,159] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-02T16:10:00+00:00 [queued]>
[2022-08-02 16:20:14,160] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 16:20:14,160] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-02 16:20:14,161] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 16:20:14,206] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-02T16:10:00+00:00
[2022-08-02 16:20:14,212] {standard_task_runner.py:51} INFO - Started process 26891 to run task
[2022-08-02 16:20:14,222] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'respond', '2022-08-02T16:10:00+00:00', '--job-id', '131', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmprj1zxoz9']
[2022-08-02 16:20:14,227] {standard_task_runner.py:76} INFO - Job 131: Subtask respond
[2022-08-02 16:20:14,324] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.respond 2022-08-02T16:10:00+00:00 [running]> on host b4dcda255553
[2022-08-02 16:20:14,434] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T16:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T16:10:00+00:00
[2022-08-02 16:20:14,435] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-02 16:20:14,485] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=respond, execution_date=20220802T161000, start_date=20220802T162014, end_date=20220802T162014
[2022-08-02 16:20:14,547] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-02 16:20:14,593] {local_task_job.py:118} INFO - Task exited with return code 0
